app/models/featured_tag.rb

```diff
@@ -59,7 +59,11 @@ def decrement(deleted_status)
   private
 
   def set_tag
-    self.tag = Tag.find_or_create_by_names(name)&.first
+    if tag.nil?
+      self.tag = Tag.find_or_create_by_names(name)&.first
+    elsif tag&.new_record?
+      tag.save!
```

Comment:
I think this `save!` is what causes the settings page to return a bogus 422 by raising a `ActiveRecord::RecordInvalid` instead of properly annotating the model with an error.

Reply:
Thank you!